client/src/templates/Challenges/components/completion-modal.tsx

```diff
@@ -155,8 +163,10 @@ class CompletionModal extends Component<
 
     if (isOpen) {
       executeGA({ event: 'pageview', pagePath: '/completion-modal' });
+      if (isCertificationProject(id)) {
```

Comment:
I think you would also need to check if the id is not in completed challenges, so it fires only when submitted for the first time.

Reply:
That would be wise.